### DISCUSSION TITLE
converting optionmenu from tkinter to customtkinter

### DISCUSSION THREAD

**Discussion by alexforecabin** [TomSchimansky/CustomTkinter#1184] [Q&A]
I am using the following code to send the selected option from the optionmenu to a label in another tab, but I am unsure as to how I can convert the optionmenu to customtkinter to get that nicely designed optionmenu? I tried just replacing with customtkinter.CTkOptionMenu but get the error; _tkinter.TclError: bad screen distance "PY_VAR0"

self.currency_input = tkinter.OptionMenu(self.tabview.tab("Form"), self.currency_var, "EUR", "USD", "SEK", command=lambda value: self.currency_label.config(text=value))

Thankful for any help, Alex.

**Comment by zSynctic** [comment]
Did you try changing all of the tkinter stuff to customtkinter if yes then try this I'm not sure if this is the solution but it might solve it if not do that thing:

```
self.currency_input = customtkinter.CTkOptionMenu(master=self.tabview.tab("Form"), variable=self.currency_var, values=["EUR", "USD", "SEK"], command = lambda value: self.currency_label.config(text=value))
```
